website/static/js/logTextParser.js

```diff
@@ -25,6 +25,12 @@ function ravenMessage (message, logObject) {
     }
 }
 
+function toRelativeUrl(url) {
+    var parser = document.createElement('a');
```

Comment:
@brianjgeiger I was referring to [http://stackoverflow.com/questions/736513/how-do-i-parse-a-url-into-hostname-and-path-in-javascript](url) when I wrote the toRelativeUrl method. It seems that it is a DOM thing. We could do some string manipulations of course but I think this is probably the most robust way as it would extract the relative path of the URL no matter what the domain name is and in what form.